Kickstarter-iOS/Views/RewardCardContainerView.swift

```diff
@@ -89,12 +89,14 @@ public final class RewardCardContainerView: UIView {
     self.viewModel.outputs.pledgeButtonHidden.observeValues { [weak self] hidden in
       guard let self = self else { return }
 
+      self.pledgeButton.isHidden = hidden
```

Comment:
Can we use the reactive binding for this? `self.pledgeButton.rac.hidden = self.viewModel.outputs.pledgeButtonHidden`